<launch>
	<node pkg="position_transpoter" type="position_transpoter" name="position_transpoter" output="screen" launch-prefix="gnome-terminal --tab --">
		<param name="uav_id" value="1" />
		<!-- 视觉丢失次数阈值 -->
		<param name="Thres_vision" value="10" />

		<!-- 跟踪到前后间隔 -->
		<param name="tracking_delta_x" value="0" />
		<param name="tracking_delta_y" value="0" />
		<param name="tracking_delta_z" value="0" />

		<!-- 相机相对于无人机偏移 -->
		<param name="camera_offset_x" value="0.1" />
		<param name="camera_offset_y" value="0" />
		<param name="camera_offset_z" value="0" />

		<!-- 起飞点 -->
		<!-- <param name="start_point_x" value="0" />
		<param name="start_point_y" value="0" />
		<param name="start_point_z" value="1" /> -->

	</node>
</launch>